src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int width;
+    private int height;
+
+    public Rectangle(String color, int height, int width) {
+        super(color);
+        this.height = height;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + getArea() + " sq. units, width: "
+                + width + " units, height: " + height + " units, color: " + getColor());

```

Comment:
The message in the `System.out.println` statement incorrectly refers to the figure as 'square' instead of 'rectangle'. Please correct the figure name in the message to match the class name.